bwpy/signal.py

```diff
@@ -0,0 +1,133 @@
+from . import mea_viewer
```

Comment:
This import can be removed

Reply:
Why?  
I use it in the `Shutter` function because it has to display by default.